Carson-Bay/strategy
# [STRAT-114] Routemodel refactor - tests and small changes (#99)

* names, variables made consistent across files

* fix 'Routes?'; renamed function

* split one last function in speedlimits

* use API key only in common file

* formatting and small adds

* function name off by one letter apparently

* corrected which variable to return

* fixed routes points builder logic

* function name correction

* Tests finished, all pass!

* Update README; routemodel tests not to be ignored

* Update yml to allow for routemodel tests on check

* Adjust some wording in yml

* fix yml ignore routemodel line

* function name changes; test changes

* unnecessary index in loop removed
